recipes-extend/uenv/uenv_1.0.bb

```diff
@@ -0,0 +1,35 @@
+# U-Boot environment for RZ SBC board
+
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
+
+SRC_URI = " \
+    file://uEnv.txt \
+    file://Readme.md \
+"
+
+S = "${WORKDIR}/sources"
+UNPACKDIR = "${S}"
```

Comment:
S = ${WORKDIR} no longer supported in Styhead, need to change the recipe to 
```
S = "${WORKDIR}/sources"
UNPACKDIR = "${S}"
```
Refs: https://docs.yoctoproject.org/next/migration-guides/migration-5.1.html#s-workdir-no-longer-supported